prototool.yaml

```diff
@@ -0,0 +1,51 @@
+# Paths to exclude when searching for Protobuf files.
+excludes:
+- vendor
+
+# Protoc directives.
+protoc:
+  # The Protobuf version to use from https://github.com/protocolbuffers/protobuf/releases.
+  # You probably want to set this to make your builds completely reproducible.
+  version: 3.7.1
+
+  # Additional paths to include with -I to protoc.
+  # By default, the directory of the config file is included,
+  # or the current directory if there is no config file.
+  includes:
+  - ../../googleapis/googleapis
+  - ../tink/proto
+
+# Lint directives.
+lint:
```

Comment:
I'd recommend starting with the `uber2` lint group and removing from there, the `uber1` lint group is the default for backwards compatibility reasons. Alternatively, there's the `google` lint group`, and an `empty` lint group that has no rules, so that you can specify lint rules with `add` instead.

Reply:
Also note we added a new `prototool lint --generate-ignores` which generates the specific `ignores` on a per id/file basis that do not pass your current files - I might recommend using this as a start. This was added in v1.5.0

Reply:
Thanks!